src/snac/client/webui/templates/page_navigation.html

```diff
@@ -113,9 +113,9 @@
             <div class="row">
                 <div class="col-md-4 footer-content">
                     <p class="text-center">
-                    <img src="{{control.snacURL}}/images/paper_dolls_art2.png"/><br/>
-                    Social Networks and Archival Context
+                        <img src="{{control.snacURL}}/images/paper_dolls_art2.png"/><br/>
                     </p>
+                    <h4 class="text-center">Social Networks and Archival Context</h4>
```

Comment:
Sneaky! I didn't notice this change :)

Reply:
@deternitydx I've removed "Cooperative". It should be fine to merge straight to dev!